src/amber/server/server.cr

```diff
@@ -44,10 +44,7 @@ module Amber
     def run
       thread_count = settings.process_count
       if Cluster.master? && thread_count > 1
-        while (thread_count > 0)
-          Cluster.fork ({"id" => thread_count.to_s})
-          thread_count -= 1
-        end
+        thread_count.times{ Cluster.fork }
```

Comment:
Small formatting issue here: _missing space_ `thread_count.times { Cluster.fork }`

Reply:
I ran `crystal tool format` on it before committing and this is how it made it...

 It looks very strange to me to put a space between the method and `()` and `{}`? When did we start doing that?

Reply:
I think is just a convention, by default `crystal tool format` _adds_ an space in between `method {` (braces) and _ignore_ spaces between `method (` (parenthesis)

Reply:
But anyway, this PR looks good, let's merge it 👍